drivers/mlx90393/include/mlx90393_params.h

```diff
@@ -38,11 +38,15 @@ extern "C" {
 #define MLX90393_PARAM_MODE             (MLX90393_MODE_BURST)
 #endif
 
+#if IS_USED(MODULE_MLX90393_INT) || DOXYGEN
 #ifndef MLX90393_PARAM_INT_PIN
 /**
  * @brief   Default interrupt pin
  */
-#define MLX90393_PARAM_INT_PIN          (GPIO_PIN(PORT_C, 8))
+#define MLX90393_PARAM_INT_PIN          .int_pin = (GPIO_PIN(PORT_C, 8)),
+#endif
+#else
+#define MLX90393_PARAM_INT_PIN
```

Comment:
So kann man das leider nicht realisieren, kein Mensch wird an der `make`-Kommandozeile ein solches Konstrukt angeben können/wollen, um den Default-Wert zu überschreiben.

```suggestion
/**
 * @brief   Default interrupt pin
 */
#ifndef MLX90393_PARAM_INT_PIN
#define MLX90393_PARAM_INT_PIN          (GPIO_PIN(PORT_C, 8))
#endif

#if IS_USED(MODULE_MLX90393_INT) || DOXYGEN
#define MLX90393_PARAM_INT             .int_pin = MLX90393_PARAM_INT_PIN,
#else
#define MLX90393_PARAM_INT
```

In der Definition von `MLX90393_PARAM_SPI` bzw. `MLX90393_PARAM_I2C` wird dann einfach grundsätzlich `MLX90393_PARAM_INT` hinzugefügt, was entweder leer ist oder ebent die Deklaration des `int_pin` Members enthält.